pkg/agent/flowexporter/exporter/util.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type tlsAddress struct{}
+
+func (addr tlsAddress) Network() string {
+	return "tcp"
+}
+func (addr tlsAddress) String() string {
+	return flowAggregatorDNSName + ":4739"
+}
+
+const (
+	CAConfigMapName       = "flow-aggregator-ca"
+	CAConfigMapKey        = "ca.crt"
+	CAConfigMapNamespace  = "flow-aggregator"
+	ClientSecretNamespace = "flow-aggregator"
+	ClientSecretName      = "flow-aggregator-client-tls"
+)
+
+func getCACert() ([]byte, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, err
+	}
+	// create clientset
+	clientSet, err := kubernetes.NewForConfig(config)
```

Comment:
get your clientset from cmd/antrea-agent/agent.go, do not build your own. AFAIK, we still try to "support" running the antrea-agent as a process, so there can be some variations when it comes to creating the clientset, and we already handle that in different places.

Reply:
not addressed yet?

Reply:
I only updated the `getClientCertKey` before. Thanks for reminding. Addressed.